packages/picongpu/package.py

```diff
@@ -69,7 +69,7 @@ class Picongpu(Package):
     depends_on('cmake@3.11.0:', type=['build', 'run'])
     depends_on('rsync', type='run')
     depends_on('util-linux', type='run', when='platform=darwin')  # GNU getopt
-    depends_on('cuda@8.0:9.2', when='backend=cuda')
```

Comment:
@psychocoderHPC does PIConGPU 0.4.3 work with CUDA 10?
Otherwise I will put 8.0-9.2 for PIConGPU 0.4.X and 9.2+ for PIConGPU `dev`/0.5.0+

Reply:
PIConGPU 0.4.3 compiles fine with CUDA 10.0 but with CUDA 10.1 and newer we are not compiling due to compiler parsing issues e.g. #758

Reply:
Excellent, that's the info I need.